VERSION.cmake

```diff
@@ -1,3 +1,3 @@
-SET(VERSION_MAJOR "2")
-SET(VERSION_MINOR "1")
-SET(VERSION_PATCH "1")
+SET(VERSION_MAJOR "4")
```

Comment:
I am a bit confused here by versioning. Which versioning this follows? ( ideally add it as comment to this file )

Thanks

Reply:
See above:
> At the same time, the version is bumped to 4.0.0 to be in sync with all the other libyui* packages (at least for now).

Reply:
OK, makes sense. In future I think it makes sense to document what exact versioning libyui follows and reference that document in each version files.